### ISSUE TITLE
[feature] add an option to fix a requirement by adding a factory that imports to this one 

### ISSUE THREAD

**Issue by warior456** [satisfactory-factories/application#258]
this would make planning from the product back to the start way easier
![image](https://github.com/user-attachments/assets/6de913d7-ea48-40a5-941c-e33bc9b9b33e)

**Comment by Maelstromeous** [comment]
@warior456 am I understanding this correctly that you want a button to create a new factory, add the item in question as a product in said factory, then automatically import it to the factory that you clicked the button on?

**Comment by warior456** [comment]
Yes (perhaps in the future being able to choose the factory to import from if it is already a product or internal product somewhere)

**Comment by Maelstromeous** [comment]
Sure why not, that might be a time saver. Unfortunately will be quite low on the priority list at this time.

**Comment by Maelstromeous** [comment]
<img width="819" height="375" alt="Image" src="https://github.com/user-attachments/assets/fc979fb4-1fd0-4191-a95c-076b407ececb" />

This has now been added to the tool! Pressing one of these buttons will either create a brand new factory with it as an item, or add it to an existing factory of which you're prompted for which one it should go ot. Thank you for the suggestion and apologies it took over a year @warior456 !
